### ISSUE TITLE
1.4.4 Resize Text - Resizing the page to 200% causes the 'Related' and 'Details' tabs to obscure the search bar of the map

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9729]
As a user reliant on zoom levels and large text to be able to perceive content, resizing the page to 200% should allow the user to access all content. No content should be obscured by other content and no content should be cut off by the edge of the page or lost entirely when the user zooms to 200%.

**Linked Work Item:**
#9493 

**Expected Result:**
All content should be perceivable and accessible at 200%.

**Actual Result:**
Resizing the page to 200% causes the 'Related' and 'Details' tabs to obscure the search bar of the map. 

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/871a68c8-2aee-4d21-82f6-5c66050db094)
